frontend/package.json

```diff
@@ -256,5 +257,8 @@
     "not dead",
     "not ie <= 11",
     "not op_mini all"
-  ]
+  ],
+  "eslintConfig": {
+    "extends": "react-app"
+  }
```

Comment:
Is this necessary for `develop`, or does it only become necessary for when we eject from CRA?

Reply:
Eject from CRA so I'll remove this.